automatically_annotate_mass2motifs/clean_library_spectra.py

```diff
@@ -0,0 +1,112 @@
+import os
+from tqdm import tqdm
+from typing import Optional
+from matchms import exporting
+import matchms.filtering as msfilters
+from matchms.logging_functions import set_matchms_logger_level
+from matchms.metadata_utils import is_valid_smiles
+from automatically_annotate_mass2motifs.bin_spectra import bin_spectra
+from automatically_annotate_mass2motifs.utils import (store_pickled_file,
+                                                      return_non_existing_file_name,
+                                                      convert_file_to_matchms_spectrum_objects)
+
+
+class FilterLibrarySpectra:
```

Comment:
Not sure if this really needs a Class, I guess a function would be fine here as well?
(but that's design details at that point...)